backend/src/main/java/com/mojprofesor/backend/repository/OpinionRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.mojprofesor.backend.repository;
+
+import com.mojprofesor.backend.entity.OpinionEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import org.springframework.data.domain.Pageable;
+import java.util.List;
+
+
+public interface OpinionRepository extends JpaRepository<OpinionEntity, Long> {
+    List<OpinionEntity> findOpinionsByProfessorID(long professorID, Pageable page);
```

Comment:
jak zmodyfikujesz encje, to tutaj będziesz musiał zmienić long na Professor, i będziesz musiał zmienić nazwę prawdopodobnie na findOpinionsByProfessor